### ISSUE TITLE
commonjs support - published to npm

### ISSUE THREAD

**Issue by michaelBenin** [jzaefferer/jquery-validation#1549]
This would be great if it had commonjs support. I would like to use this with browserify.

**Comment by antitoxic** [comment]
+1 . Was wandering why the project was on NPM without commonjs support.

I've seen this to be a common approach
```
function (root, factory) {

  /* CommonJS */
  if (typeof module == 'object' && module.exports) module.exports = factory()

  /* AMD module */
  else if (typeof define == 'function' && define.amd) define(factory)

  /* Browser global */
  else root.Spinner = factory()
}(this, function () {....})
```

**Comment by Atsman** [comment]
+1

**Comment by jzaefferer** [comment]
The build currently doesn't include "commonjs" exports, only [define and globals](https://github.com/jzaefferer/jquery-validation/blob/1.14.0/dist/jquery.validate.js#L9-L15). 

Could one of you put together a patch (and send as pull request) to include the `module.exports` part? Relevant code should be here: https://github.com/jzaefferer/jquery-validation/blob/f8e4be597967d0971d0f76912f183cdf1cda46b5/Gruntfile.js#L25-L31

**Comment by Nettsentrisk** [comment]
+1

**Comment by gregorej** [comment]
+1

**Comment by gregorej** [comment]
Just created pull request for that: https://github.com/jzaefferer/jquery-validation/pull/1597

**Comment by codeclown** [comment]
+1

**Comment by staabm** [comment]
landed with #1621
